pkg/webhook/pod/tikv_deleter.go

```diff
@@ -46,12 +46,23 @@ func (pc *PodAdmissionControl) admitDeleteTiKVPods(payload *admitPayload) *admis
 		return util.ARFail(err)
 	}
 
+	tc, ok := payload.controller.(*v1alpha1.TidbCluster)
+	if !ok {
+		err := fmt.Errorf("tikv pod[%s/%s]'s controller is not tidbcluster,forbid to be deleted", namespace, name)
+		return util.ARFail(err)
+	}
```

Comment:
Why adding this? It's not reasonable to block the deletion of non-tidbcluster managed Pods.